### PR TITLE
[NETBEANS-1144] Recreate predefined_vars.zip

### PR DESCRIPTION
**Pull request by junichi11**
https://issues.apache.org/jira/browse/NETBEANS-1144

Recreate predefined_vars.zip because it is licensed under LGPL-2.1 so far.

- Create files from https://secure.php.net/manual/en/reserved.variables.php
- The license is CC-BY 3.0 https://secure.php.net/manual/en/copyright.php
- CC-BY 3.0 is listed in category-b: https://www.apache.org/legal/resolved.html#category-b

### CODE REVIEW

File: php.editor/external/predefined_vars-license.txt [hunk 1]
```diff
@@ -1,505 +1,65 @@
-License: LGPL-2.1
+Name: PHP Documentation
+Version: 1.0
+License: CC-BY 3.0
+Description: Documentation for Predefined Variables
+Origin: https://secure.php.net/manual/en/
+Files: predefined_vars.zip
```
[the remark this answers is not in the document]
Comment (junichi11) [reply php.editor/external/predefined_vars-license.txt]: Can we remove "Version"? or Do we have to add it?

File: php.editor/external/predefined_vars-license.txt [hunk 2]
```diff
@@ -1,505 +1,65 @@
-License: LGPL-2.1
+Name: PHP Documentation
+Version: 1.0
+License: CC-BY 3.0
```
[the remark this answers is not in the document]
Comment (junichi11) [reply php.editor/external/predefined_vars-license.txt]: I see. Will do it. Thanks!

File: php.editor/external/predefined_vars-license.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (matthiasblaesing) [reply php.editor/external/predefined_vars-license.txt]: Version is a required header.

[the remark this answers is not in the document]
Comment (junichi11) [reply php.editor/external/predefined_vars-license.txt]: OK, I'll rename it, then upload it again. Thanks!

[the remark this answers is not in the document]
Comment (junichi11) [reply php.editor/external/predefined_vars-license.txt]: Fixed.

File: php.editor/external/predefined_vars-license.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (junichi11) [reply php.editor/external/predefined_vars-license.txt]: Fixed.

**Comment by junichi11** [comment]
Sorry, will fix predefined_vars-license.txt again. (I've deleted not renamed)

**Comment by geertjanw** [comment]
Thanks!

**Pull request merged**